packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js

```diff
@@ -2,8 +2,9 @@ import { DOC_URL } from './constants';
 
 export default class KubernetesCtrl {
   /* @ngInject */
-  constructor($scope) {
+  constructor($scope, ovhFeatureFlipping) {
     this.$scope = $scope;
+    this.ovhFeatureFlipping = ovhFeatureFlipping;
```

Comment:
```suggestion
  constructor($scope) {
    this.$scope = $scope;
```

You don't use in this controller